nifi-nar-bundles/nifi-cdc/nifi-cdc-mysql-bundle/nifi-cdc-mysql-processors/src/main/java/org/apache/nifi/cdc/mysql/processors/CaptureChangeMySQL.java

```diff
@@ -263,6 +263,17 @@
             .addValidator(StandardValidators.BOOLEAN_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor INCLUDE_SCHEMA_CHANGES = new PropertyDescriptor.Builder()
+            .name("capture-change-mysql-include-schema-changes")
+            .displayName("Include Schema Change Events")
+            .description("Specifies whether to emit events corresponding to a schema change event (ALTER TABLE, e.g.) in the binary log. Set to true if the schema change events are "
+                    + "desired/necessary in the downstream flow, otherwise set to false, which suppresses generation of these events and can increase flow performance.")
```

Comment:
Now that we're including things like truncate, I agree.  However we refer to "schema change" many times in the CDC code. I wrote up NIFI-3735 to cover the change in terminology across the CDC code. In the meantime however, I will just update this property to have the correct documentation.